### ISSUE TITLE
Issue during Docker start

### ISSUE THREAD

**Issue by jmerifjKriwe** [tillsteinbach/WeConnect-mqtt#48]
Hi,

I want to switch my WeConnect-mqtt to a docker container. But it's not starting:

```Traceback (most recent call last):,
  File "/usr/local/lib/python3.9/site-packages/urllib3/connection.py", line 174, in _new_conn,
    conn = connection.create_connection(,
  File "/usr/local/lib/python3.9/site-packages/urllib3/util/connection.py", line 73, in create_connection,
    for res in socket.getaddrinfo(host, port, family, socket.SOCK_STREAM):,
  File "/usr/local/lib/python3.9/socket.py", line 953, in getaddrinfo,
    for res in _socket.getaddrinfo(host, port, family, type, proto, flags):,
socket.gaierror: [Errno -3] Try again,
,
During handling of the above exception, another exception occurred:,
,
Traceback (most recent call last):,
  File "/usr/local/lib/python3.9/site-packages/urllib3/connectionpool.py", line 699, in urlopen,
    httplib_response = self._make_request(,
  File "/usr/local/lib/python3.9/site-packages/urllib3/connectionpool.py", line 382, in _make_request,
    self._validate_conn(conn),
  File "/usr/local/lib/python3.9/site-packages/urllib3/connectionpool.py", line 1010, in _validate_conn,
    conn.connect(),
  File "/usr/local/lib/python3.9/site-packages/urllib3/connection.py", line 358, in connect,
    conn = self._new_conn(),
  File "/usr/local/lib/python3.9/site-packages/urllib3/connection.py", line 186, in _new_conn,
    raise NewConnectionError(,
urllib3.exceptions.NewConnectionError: <urllib3.connection.HTTPSConnection object at 0xb5bdf1c0>: Failed to establish a new connection: [Errno -3] Try again,
,
During handling of the above exception, another exception occurred:,
,
Traceback (most recent call last):,
  File "/usr/local/lib/python3.9/site-packages/requests/adapters.py", line 439, in send,
    resp = conn.urlopen(,
  File "/usr/local/lib/python3.9/site-packages/urllib3/connectionpool.py", line 783, in urlopen,
    return self.urlopen(,
  File "/usr/local/lib/python3.9/site-packages/urllib3/connectionpool.py", line 783, in urlopen,
    return self.urlopen(,
  File "/usr/local/lib/python3.9/site-packages/urllib3/connectionpool.py", line 783, in urlopen,
    return self.urlopen(,
  File "/usr/local/lib/python3.9/site-packages/urllib3/connectionpool.py", line 755, in urlopen,
    retries = retries.increment(,
  File "/usr/local/lib/python3.9/site-packages/urllib3/util/retry.py", line 574, in increment,
    raise MaxRetryError(_pool, url, error or ResponseError(cause)),
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='login.apps.emea.vwapps.io', port=443): Max retries exceeded with url: /authorize?nonce=jrQusoT7mxW9NOoC&redirect_uri=weconnect://authenticated (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0xb5bdf1c0>: Failed to establish a new connection: [Errno -3] Try again')),
,
During handling of the above exception, another exception occurred:,
,
Traceback (most recent call last):,
  File "/usr/local/lib/python3.9/site-packages/weconnect_mqtt/weconnect_mqtt_base.py", line 208, in main,
    mqttCLient.connectWeConnect(username=username, password=password, maxAgePictures=args.pictureCache),
  File "/usr/local/lib/python3.9/site-packages/weconnect_mqtt/weconnect_mqtt_base.py", line 276, in connectWeConnect,
    self.weConnect = weconnect.WeConnect(username=username, password=password, updateAfterLogin=False, updateCapabilities=self.updateCapabilities,,
  File "/usr/local/lib/python3.9/site-packages/weconnect/weconnect.py", line 205, in __init__,
    self.login(),
  File "/usr/local/lib/python3.9/site-packages/weconnect/weconnect.py", line 270, in login,
    tryLoginResponse: requests.Response = self.__session.get(tryLoginUrl, allow_redirects=False),
  File "/usr/local/lib/python3.9/site-packages/requests/sessions.py", line 555, in get,
    return self.request('GET', url, **kwargs),
  File "/usr/local/lib/python3.9/site-packages/requests/sessions.py", line 542, in request,
    resp = self.send(prep, **send_kwargs),
  File "/usr/local/lib/python3.9/site-packages/requests/sessions.py", line 655, in send,
    r = adapter.send(request, **kwargs),
  File "/usr/local/lib/python3.9/site-packages/requests/adapters.py", line 516, in send,
    raise ConnectionError(e, request=request),
requests.exceptions.ConnectionError: HTTPSConnectionPool(host='login.apps.emea.vwapps.io', port=443): Max retries exceeded with url: /authorize?nonce=jrQusoT7mxW9NOoC&redirect_uri=weconnect://authenticated (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0xb5bdf1c0>: Failed to establish a new connection: [Errno -3] Try again')),
,
During handling of the above exception, another exception occurred:,
,
Traceback (most recent call last):,
  File "/usr/local/bin/weconnect-mqtt", line 8, in <module>,
    sys.exit(main()),
  File "/usr/local/lib/python3.9/site-packages/weconnect_mqtt/weconnect_mqtt_base.py", line 245, in main,
    mqttCLient.disconnect(),
  File "/usr/local/lib/python3.9/site-packages/weconnect_mqtt/weconnect_mqtt_base.py", line 271, in disconnect,
    disconectPublish.wait_for_publish(),
  File "/usr/local/lib/python3.9/site-packages/paho/mqtt/client.py", line 362, in wait_for_publish,
    raise RuntimeError('Message publish failed: %s' % (error_string(self.rc))),
RuntimeError: Message publish failed: The client is not currently connected.

**Comment by jmerifjKriwe** [comment]
I'm using the same credentials as on my current installation.

**Comment by tillsteinbach** [comment]
It looks like it is not able to resolve the address. Can you check that the container really has access to the internet?

**Comment by tillsteinbach** [comment]
It looks like the VW servers also have a problem right now.

**Comment by jmerifjKriwe** [comment]
But the commandline version (outside the container) is still working, strange.

Any idea how I can connect to the container and check if it has access to the internet? Cause due to the error the container is stopping/crashing right after the start.

**Comment by jmerifjKriwe** [comment]
Ok, ok. Finally I managed to be very quick and be able to do some tests before the container crashes.
So access to the internet is available but DNS resolution is not working. Which is wired because my other container in the same network don't have this issue.

**Comment by jmerifjKriwe** [comment]
Fixed it buy adding 8.8.8.8 as DNS server in the container settings. Don't why this is necessary for this container but now it works.
